src/main/webapp/js/user-page-loader.js

```diff
@@ -42,9 +42,11 @@ function showMessageFormIfViewingSelf() {
             loginStatus.username == parameterUsername) {
           const messageForm = document.getElementById('message-form');
           messageForm.classList.remove('hidden');
+          document.getElementById('about-me-form').classList.remove('hidden');
+          }
         }
       });
-}
```

Comment:
I think this is still needed to match the brace on line 35.

Reply:
oops, thank you!